Framework/DataHandling/src/LoadMuonNexus1.cpp

```diff
@@ -111,6 +111,33 @@ void LoadMuonNexus1::exec() {
     g_log.warning() << "Error while loading the FirstGoodData value: " << e.what() << "\n";
   }
 
+  try {
+    NXInfo infoResolution = entry.getDataSetInfo("resolution");
+    NXInt counts = root.openNXInt("run/histogram_data_1/counts");
+    std::string lastGoodBin = counts.attributes("last_good_bin");
```

Comment:
there is no last_good_bin entry in the nxs 1 files. So just set the last good value to be the largest x value. And add a comment in the code explaining that there isnt a value in the files.